pydis_site/apps/api/models/bot/tag.py

```diff
@@ -93,7 +93,7 @@ def validate_tag_embed_author(author):
             validator(value)
 
 
-def validate_tag_embed(embed):
+def validate_tag_embed(embed: Dict[str, Union[str, List[dict], dict]]) -> None:
```

Comment:
This JSON type is not comprehensive. For the sake of simplicity, I think it will be better to use `Union[List, Dict]`,  or even just `Dict` if we follow the line of thinking that the Discord API does not return a list for Embed objects.

Reply:
I'm not really familiar with how Django's bits go together, isn't this validating what's coming out of our DB rather than what's coming from Discord?

Reply:
We're validating the data that's being received by the API, which obviously comes from the bot and not directly from Discord. When it comes to embeds, it's a `json` representation of the `to_dict` method of `discord.Embed` from discord.py.

Reply:
I think `Dict` is good enough here and I agree with @SebastiaanZ we shouldn't over-think this. But along the the lines of the other comment I made, I may as well make the point that since this is a validation function and it also does `isinstance(..., Mapping)`, I think a more technically correct type would be `Any` or something along those lines.

Reply:
So...

```py
def validate_tag_embed(embed: Any) -> None:
```
?

Reply:
I'm fine with that. If you or anyone else doesn't object then sure.

Reply:
@jchristgit what are your thoughts?